### PR TITLE
Bump pulldown_cmark to version 0.6.0

### PR DESCRIPTION
**Pull request by marcusklaas**
**IMPORTANT: Please do not create a Pull Request adding a new feature without discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the feature requests section.

Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [x] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [x] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

File: components/rendering/src/markdown.rs [hunk 1]
```diff
@@ -33,12 +33,12 @@ pub struct Rendered {
 struct HeaderRef {
     start_idx: usize,
     end_idx: usize,
-    level: i32,
+    level: u32,
     id: Option<String>,
 }
 
 impl HeaderRef {
```
Comment (Keats) [inline components/rendering/src/markdown.rs]: can you rename all those Header structs/variable to Heading to be consistent?

File: components/rendering/Cargo.toml [hunk 2]
```diff
@@ -6,7 +6,7 @@ authors = ["Vincent Prouillet <prouillet.vincent@gmail.com>"]
 [dependencies]
 tera = { version = "1.0.0-beta.10", features = ["preserve_order"] }
 syntect = "=3.2.0"
-pulldown-cmark = "0.5"
+pulldown-cmark = "0.6"
```
Comment (Keats) [inline components/rendering/Cargo.toml:9]: I think it might be used in some other sub crates as well? Can't remember

**Review by Keats — CHANGES_REQUESTED** [review]

File: components/rendering/Cargo.toml (hunk 2 above)
Comment (marcusklaas) [reply components/rendering/Cargo.toml:9]: You are right! I'll update those too.

File: components/rendering/src/markdown.rs (hunk 1 above)
Comment (marcusklaas) [reply components/rendering/src/markdown.rs]: Can do!

**Comment by marcusklaas** [comment]
I have updated the PR.

**Comment by Keats** [comment]
Thanks a lot for that and for your work on pulldown_cmark!

**Pull request merged**